docs/core/project-sdk/msbuild-props.md

```diff
@@ -443,6 +443,36 @@ Restoring a referenced package installs all of its direct dependencies and all t
 </PropertyGroup>
 ```
 
+## Run properties
+
+The following properties are used for launching an app during development in Visual Studio or with the [`dotnet run`](../tools/dotnet-run.md) command:
+
+- [RunArguments](#runarguments)
+- [RunWorkingDirectory](#runworkingdirectory)
+
+### RunArguments
+
+The `RunArguments` property defines the arguments that are passed to the app when it is run.
+
+```xml
+<PropertyGroup>
+  <RunArguments>-mode dryrun</RunArguments>
+</PropertyGroup>
+```
+
+> [!TIP]
+> You can specify additional arguments to be passed to the app by using the [`--` option for `dotnet run`](../tools/dotnet-run.md#options).
+
+### RunWorkingDirectory
+
+The `RunWorkingDirectory` property defines the working directory for the application process to be started in.
```

Comment:
@dsplaisted I'd like to put what the default working directory is, but I couldn't figure it out. All I can see is that if `RunWorkingDirectory` is empty, it defaults to `StartWorkingDirectory`, but I couldn't find out where that is set. Can you help?

`<RunWorkingDirectory Condition="'$(RunWorkingDirectory)' == ''">$(StartWorkingDirectory)</RunWorkingDirectory>`

Reply:
I don't think it's set by default.  It looks like the default is to use `OutDir`:

https://github.com/dotnet/msbuild/blob/a2c42ce9fc8d6fbd4a8cd600073803b8687c9341/src/Tasks/Microsoft.Common.CurrentVersion.targets#L1094-L1095